README.md

```diff
@@ -41,9 +43,24 @@ The command-line tool is called `jj` for now because it's easy to type and easy
 to replace (rare in English). The project is called "Jujutsu" because it matches
 "jj".
 
-If you have any questions, please join us on Discord
+Jujutsu is relatively young, with lots of work to still be done. If you have any
+questions, or want to talk about future plans, please join us on Discord
 [![Discord](https://img.shields.io/discord/968932220549103686.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/dkmfj3aGQN)
-or start a [GitHub Discussion](https://github.com/martinvonz/jj/discussions).
+or start a [GitHub Discussion](https://github.com/martinvonz/jj/discussions); the
+developers monitor both channels.
+
+> [!IMPORTANT]
+> Jujutsu is an **experimental version control system**. While Git compatibility
+> is stable, and most developers use it daily for all their needs, there may
+> still be performance pitfalls, suboptimal UX, and feature gaps that make it
```

Comment:
FYI, once we've enabled tree-level conflicts by default, I think we can say that we've fixed all the serious performance bugs (that I can think of). I'll try to remember to update this sentence then,